src/components/decimal/decimal.component.tsx

```diff
@@ -179,6 +158,30 @@ export const Decimal = React.forwardRef(
       [getSeparator, isNaN, l, locale, precision]
     );
 
+    const emptyValueWithPrecision =
+      precision === 0 ? "0" : formatValue(`0.${"0".repeat(precision)}`);
```

Comment:
**question** (non-blocking) - is there a good reason to build up the precision value manually like this, instead of just using `(0).toFixed(precision)`? (From trying it in the console you unfortunately need those ugly parentheses around the 0, otherwise JS thinks the `.` is a decimal point rather than for object property access...)

It works correctly for precision 0 too so you wouldn't even need the ternary. And you probably don't even need the separate const as it's only used once and the expression will become simple enough.

Reply:
I seem to remember trying `.toFixed()` back in 2020 when I refactored Decimal and I seem to remember it rounded up values when we didn't want it to. Might have remember incorrectly though... 

In this case, I don't think accidentally rounding up would be an issue as it's just `0`'s. I'll give it a try and see if the tests pass 😃